### PR TITLE
Add readnoise monitor bokeh plots

### PR DESCRIPTION
**Pull request by bsunnquist**
Adds bokeh plots/webpages for the readnoise monitor

### CODE REVIEW

**Comment by bsunnquist** [comment]
@bourque This is all ready for review - the plot webpages should show like below:
Screen Shot 2020-10-02 at 4 23 26 PM

One assumption I made was that users will be able to display png images stored in our central storage directories (i.e. see line 133 in `monitor_readnoise_bokey.py`). I believe this is possible since we display the unlooked png images, but let me know if this will be an issue.

File: jwql/website/apps/jwql/monitor_pages/monitor_readnoise_bokeh.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+"""This module contains code for the readnoise monitor Bokeh plots.
```
Comment (bourque) [inline jwql/website/apps/jwql/monitor_pages/monitor_readnoise_bokeh.py:1]: This is well organized and well commented, great work!

**Review by bourque — APPROVED** [review]
Thank you @bsunnquist, this looks great!  And it works on me end when I test it locally. This will be ready to merge pending the resolution of those merge conflicts.

**Comment by bsunnquist** [comment]
@bourque Should be good to go now. I just merged those new `template.py` changes into this branch, and things are still working properly.

**Comment by bourque** [comment]
@bsunnquist Awesome, thanks!

**Pull request merged**